pkgs/development/tools/alloy/default.nix

```
@@ -50,20 +50,20 @@ let generic = { major, version, src }:
   };
 
 in rec {
-  alloy4 = let version = "4.2_2015-02-22"; in generic {
-    major = "4";
+  alloy5 = let version = "5.1.0"; in generic {
+    major = "5";
     inherit version;
```

```suggestion
  alloy5 = generic rec {
    version = "5.1.0"
```
This can be replaced with ``lib.versions.major version``.